protocol/avdl/chat1/common.avdl

```diff
@@ -353,17 +352,18 @@
     gregor1.Time composeTime;
   }
 
-  @mpackkey("em") @jsonkey("em")
   record MsgEphemeralMetadata {
     @mpackkey("l") @jsonkey("l")
     gregor1.DurationSec lifetime; // used to computed etime
     @mpackkey("g") @jsonkey("g")
     keybase1.EkGeneration generation;
     @mpackkey("u") @jsonkey("u")
     union { null, string } explodedBy;
+    // Non-null for TopicType BOT, specifies which member is the bot recipient
+    // to select appropriate keys
+    union { null, gregor1.UID } botUID;
   }
 
-  @mpackkey("pi") @jsonkey("pi")
```

Comment:
note these decorators on the record type were erroneous, must be set on the field within the record. the json output respects this but go does not, removed these to prevent confusion